client/src/AppSecondary.jsx

```diff
@@ -21,6 +21,19 @@ const AppContainer = styled.div`
   height: 600px;
 `;
 
+const CheckoutOnlyIndicator = styled.div`
+  position: absolute;
+  z-index: 109;
+  left: ${props => (props.checkoutOnlyX.toString() + 'px')};
+  top: ${props => (props.checkoutOnlyY.toString() + 'px')};
```

Comment:
It seems like positioning could be accomplished without state. Maybe something like this?
https://www.w3schools.com/css/css_tooltip.asp